.github/dockerursim/safety.conf.UR5

```diff
@@ -0,0 +1,101 @@
+# Beware: This file is auto-generated from PolyScope.
+# NOTE: The SafetyParameters section is protected by a CRC checksum, please use the supplied tool
+
+## SafetyParameters ##
+[NormalModeSafetyLimits]
+maxTcpSpeed = 1.5
+maxForce = 150.0
+maxElbowSpeed = 1.5
+maxElbowForce = 150.0
+maxStoppingDistance = 0.5
+maxStoppingTime = 0.4
+maxPower = 300.0
+maxMomentum = 25.0
+maxJointSpeed = [3.3415926, 3.3415926, 3.3415926, 3.3415926, 3.3415926, 3.3415926]
+minJointPosition = [6.2308254, 6.2308254, 6.2308254, 6.2308254, 6.2308254, 6.2308254]
+maxJointPosition = [0.05235988, 0.05235988, 0.05235988, 0.05235988, 0.05235988, 0.05235988]
```

Comment:
Is this correct?

Reply:
No! Thanks for spotting that! I've been playing around with the safety settings before exporting this file. That should be changed.

Reply:
Perhaps import a vanilla (ie: just extracted from the ursim archive) version of the file?

Reply:
Update: Yes, it is correct. I just downloaded it from a robot with default safety configuration.

Reply:
> Perhaps import a vanilla (ie: just extracted from the ursim archive) version of the file?

In fact in the archive this file is empty so that the user has to confirm it first. This is why we actually do that step.